brave/brave-core
# Revert `Implement "reduce language fingerprinting"` and follow ups

Revert `Implement "reduce language fingerprinting"` and follow ups
Fixes https://github.com/brave/brave-browser/issues/23093

Reopens https://github.com/brave/brave-browser/issues/20096
Reopens https://github.com/brave/brave-browser/issues/816

<hr>

Reverts https://github.com/brave/brave-core/pull/12952
Which fixed https://github.com/brave/brave-browser/issues/22177

Reverts https://github.com/brave/brave-core/pull/12942
Which fixed https://github.com/brave/brave-browser/issues/21985

Reverts https://github.com/brave/brave-core/pull/13807
Which fixed https://github.com/brave/brave-browser/issues/23489

Reverts https://github.com/brave/brave-core/pull/13205
Which fixed https://github.com/brave/brave-browser/issues/22605

Reverts https://github.com/brave/brave-core/pull/12234
Which fixed https://github.com/brave/brave-browser/issues/20096 and https://github.com/brave/brave-browser/issues/816

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Visit brave://settings/shields
2. Verify the setting `Prevent sites from fingerprinting me based on my language preferences` was removed (if present, it would be above `Trackers & ads blocking`)
3. Try out some of the sites called out in https://github.com/brave/brave-browser/issues/23093 and make sure font looks correct
